inz-demo/id-card/Issues/address/description.xml

```diff
@@ -0,0 +1,34 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo ID Card Address</en>
+        <nl>Demo ID Card Address</nl>
+    </Name>
+    <ShortName>
+        <en>Address</en>
+        <nl></nl>
+    </ShortName>
+    <SchemeManager>inz-demo</SchemeManager>
+    <IssuerID>id-card</IssuerID>
+    <CredentialID>address</CredentialID>
+    <Description>
+        <en>Full address</en>
+        <nl>Full address</nl>
+    </Description>
+    <ShouldBeSingleton>true</ShouldBeSingleton>
```

Comment:
I think that we do not need to keep singleton for Address. One can have more than one address on the wallet.